tests/nodeUnit.js

```diff
@@ -69,7 +69,45 @@ define(function(require) {
           'dist/public/index.html',
           'dist/public/manifest.json',
           'dist/public/offline-worker.js',
-          'dist/public/status.json'
+          'dist/public/status.json',
+          'dist/public/app-manifest.html',
```

Comment:
Yup, +1 on making this list auto-generated. I can add that to my list of TODOs and get to it before the end of next week.